gen/gen.go

```diff
@@ -119,7 +117,7 @@ func main() {
 		// method params and returns
 		//paych.ConstructorParams{}, // Aliased from v0
 		paych.UpdateChannelStateParams{}, // Changed in v7
-		paych.SignedVoucher{}, // Changed in v7
+		paych.SignedVoucher{},            // Changed in v7
```

Comment:
Can you alias these (and below) below onto their v7 definitions?

Reply:
Yes, thank you for the catch!